apply-patches.sh

```diff
@@ -19,8 +19,9 @@ cd $ORIG_DIR
 for NAPP in $(ls -1 patches/napps/); do
 	echo $NAPP
 	NAPP_DIR=$(python3 -m pip show $NAPP | grep Location | awk '{print $NF}')
+	test -z "$NAPP_DIR" && NAPP_DIR=/src/$(echo $NAPP | tr '_' '-')
 	cd $NAPP_DIR
-	for PATCH in $(find $ORIG_DIR/patches/napps/$NAPP/ -type f -name '*.patch'); do
+	for PATCH in $(find $ORIG_DIR/patches/napps/$NAPP/ -type f -name '*.patch' | sort); do
```

Comment:
This will guarantee the patches are executed in alphabetical order according to their names (that is why we prefix them with xxx_)